spring-cloud-skipper/spring-cloud-skipper-server-core/pom.xml

```diff
@@ -279,28 +283,6 @@
 		</plugins>
 	</build>
 	<profiles>
-		<profile>
-			<id>full</id>
-			<build>
-				<plugins>
-					<plugin>
-						<groupId>org.apache.maven.plugins</groupId>
-						<artifactId>maven-surefire-plugin</artifactId>
-						<configuration>
-							<argLine>--add-opens java.base/java.util=ALL-UNNAMED</argLine>
-							<threadCount>1</threadCount>
-							<forkCount>1</forkCount>
-							<failIfNoTests>true</failIfNoTests>
-							<skipTests>false</skipTests>
-							<includes>
-								<include>**/*Documentation.java</include>
-								<include>**/*Tests.java</include>
```

Comment:
We probably want to include the `**/*Tests.java` to the docs plugin below.   WDYT?

Reply:
I've changed to the surefire in `docs` profile to use the `package` phase which means `test` phase executes normal unit tests and `package` phase generates the snippets. 
When `-DskipTests` is present the normal test phase is skipped. When -Pdocs is not provides then the snippets are not generated.